core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -205,7 +205,7 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         <md-card ng-repeat="exploration in explorationsList | orderBy:sortByFunction:isCurrentSortDescending track by exploration.id"
                  class="oppia-activity-summary-tile oppia-dashboard-card-view-item protractor-test-exploration-dashboard-card"
                  is-private="exploration.status === 'private'">
-          <a ng-href="/create/<[exploration.id]>">
+          <a ng-href="/create/<[exploration.id]>#/gui/Introduction">
```

Comment:
Re first paragraph: actually, that's not necessarily the case. It's possible to change the initial state name.

Re second paragraph: I added a comment inline.